lenses/mke2fs.aug

```diff
@@ -95,24 +130,12 @@ let defaults = IniFile.record defaults_title
  * Group:                 FS_TYPES SECTION
  *************************************************************************)
```

Comment:
Is there a reason for reorganizing these entries under common?

Reply:
You mean all the content of `fs_types_entry`? As I wrote in the commit message, the documentation says that they are allowed both in `[defaults]`, and for each tag in `[fs_types]`. Since `common_entry` represents exactly this, then this is why they were moved.

Reply:
Ah right, sorry I missed that!